compiler_opt/rl/compilation_runner.py

```diff
@@ -331,15 +285,15 @@ def signaler():
 
   def collect_data(
       self,
-      file_paths: Tuple[str, ...],
+      module_spec: adt.ModuleSpec,
```

Comment:
`adt` is too generic - abstract data types? How about a `corpus` module; then we can add a corpus reader encapsulate a source of `ModuleSpec`s. That will, among other things, simplify how we know it is or not thinlto, because that is a corpus-wide property.